packages/vat-data/src/far-class-utils.js

```diff
@@ -19,13 +19,14 @@ import {
 
 // TODO interfaceGuard type https://github.com/Agoric/agoric-sdk/issues/6206
 /**
- * @template A,S,T
+ * @template {(...args: any) => any} I init state function
```

Comment:
Nit: not strictly necessary, but conventional.
```suggestion
 * @template {(...args: any[]) => any} I init state function
```